x/leveragelp/keeper/position.go

```diff
@@ -416,3 +421,23 @@ func (k Keeper) GetPositionWithId(ctx sdk.Context, positionAddress sdk.Address,
 	k.cdc.MustUnmarshal(res, &position)
 	return &position, true
 }
+
+// func (k Keeper) MigrateKeys(ctx sdk.Context) {
```

Comment:
@amityadav0 any reason to keep this function commented?

Reply:
yes, currently we only avoid the error while loading and not entirely delete the corrupted key, value
This is done to ensure we don't delete everything.

I was thinking to uncomment it in later upgrade once we are sure the issue is fixed

Reply:
@amityadav0 yep make sense could you add a `FIXME` or `TODO` comment above the commented function with this context to make sure we keep that in mind thanks

Reply:
thanks, added